src/lib/features/metrics/last-seen/last-seen-store.ts

```diff
@@ -48,7 +48,7 @@ export default class LastSeenStore implements ILastSeenStore {
     async setLastSeen(data: LastSeenInput[]): Promise<void> {
         try {
             const inserts = prepareLastSeenInput(data);
-
+            console.log('inserts', inserts);
```

Comment:
we can remove it

Reply:
[7f030b8](https://github.com/Unleash/unleash/pull/7923/commits/7f030b8f0f21fea72abdc026120f4dfe95824d7f)